src/matchers/toLog/predicate.js

```diff
@@ -0,0 +1,21 @@
+import { equals } from 'expect/build/jasmine_utils';
+
+export default (fn, expected, consoleMethod) => {
+  const spy = jest.spyOn(console, consoleMethod);
```

Comment:
if we set up the spy here I can imagine we could incure larger performance implications than if it wasn't set at run time, but ill comment on that in my main response 😇